### DISCUSSION TITLE
Methods to design the left-pane navigation

### DISCUSSION THREAD

**Discussion by anindyo** [google/docsy#1497] [Q&A]
Designing the left-pane navigation of a Hugo site is done via front-matter `weight` values. Do we have any alternatives to that method? This approach becomes very cumbersome over time. For example, defining the structure within the config.toml file would be much easier than dealing with the `weight` values of 1000s of topic files. Other SSGs provide a far simpler mechanism for left-pane navigation design--via an external YAML (Mkdocs) or config.json (Docusaurus).

**Comment by anindyo** [comment]
Checking in again. Any ideas or suggestions?

**Comment by emckean** [comment]
This is an interesting idea, thank you!

**Reply by anindyo** [reply]
@emckean Thanks for your comment! Do you know if this is under consideration by the Hugo/Docsy team?
